Frontend/src/Components/Enums/Enums.js

```diff
@@ -1 +1,40 @@
+export const Categories = {
+  WOMENSWEAR: "womenswear",
+  MENSWEAR: "menswear",
+};
 
+export const Subcategories = {
```

Comment:
hmm so i think for here i would just have a subcategory enum that looks more like your category enum, so like 
```
TSHIRT: 'tshirt' 
TROUSERS: 'trousers'
....
```
and then i would have two constants in this file, one for womenswear subcategories and one for menswear subcategories:
```
export const WOMENSWEAR_SUBCATEGORIES= [Subcategory.TOPS, Subcategory.TSHIRTS, Subcategory.JEANS, Subcategory.DRESSES, Subcategory.SKIRTS]
export const MENSWEAR_SUBCATEGORIES = [Subcategory.TOPS, Subcategory.TSHIRTS, Subcategory.JEANS, Subcategory.TROUSERS, Subcategory.SHORTS]
```
then we have a more straightforward map of these constants in infosection.jsx in the next commit